packages/metamask/src/index.ts

```diff
@@ -78,25 +66,24 @@ function metamask({
 
         const { createEIP1193Provider, MetaMaskSDKConstructor } = imports
 
-        createInstance = createEIP1193Provider
```

Comment:
Now that we arent using `createEIP1193Provider` can it be removed from the imports object and function above?